1-js/13-modules/01-modules-intro/article.md

```diff
@@ -83,7 +83,7 @@ sayHi('John'); // Hello, John!
 
 외부에 공개하려는 모듈은 `export` 해야 하고, 내보내진 모듈을 가져와 사용하려면 `import` 해줘야 합니다.
 
-`index.html`가 아닌 `hello.js` 에서 `user.js`를 가져와 `user`를 사용해봅시다.
+ 전역변수를 대신하여 `hello.js`에 `user.js`를 가져와 필요한 기능을 얻을 수 있습니다.
```

Comment:
원문 내용입니다. 내용이 바뀐 것 같아 수정했어요.
```
So we should import user.js into hello.js and get the required functionality from it instead of
relying on global variables.
```